src/main/java/seedu/jarvis/logic/parser/DeleteCommandParser.java

```diff
@@ -18,43 +25,82 @@
 public class DeleteCommandParser implements Parser<DeleteCommand> {
 
     private static final Logger logger = LogsCenter.getLogger(DeleteCommandParser.class);
+    private static final String EMPTY_DELETE_COMMAND = "Parameters should not be empty for delete commands; enter id.";
```

Comment:
Perhaps the message can be clearer? e.g. "Please enter an index number after the command. e.g. delete -mc 1"

Reply:
Done! Thanks for the input, I appreciate it!